src/wallet/rpcdump.cpp

```diff
@@ -700,7 +718,7 @@ UniValue z_importkey(const UniValue& params, bool fHelp)
 
     // Sapling support
     auto addResult = boost::apply_visitor(
-        AddSpendingKeyToWallet(pwalletMain, Params().GetConsensus(), 1, false), spendingkey);
+        AddSpendingKeyToWallet(pwalletMain, Params().GetConsensus(), 1, boost::none, boost::none, false), spendingkey);
```

Comment:
You should be able to set the four new parameters as defaults in the `operator()` declaration, so that you don't have to specify them in these other locations. Then again, we only use this visitor in these two locations, so maybe explicit is better...